tests/rhui3_tests_lib/util.py

```diff
@@ -140,3 +140,9 @@ def wildcard(hostname):
         hostname_particles[0] = "*"
         return ".".join(hostname_particles)
 
+    @staticmethod
+    def esc_parentheses(name):
+        '''
+        helper method to escape parentheses so they can be safely used inside regular expressions in Expect methods
```

Comment:
@RadekBiba too long line, and there are others.

Reply:
It's the longest comment, but it's as descriptive as it can be. It doesn't break anything AFAICT, so I'd like to keep it.

Reply:
it's a comment string =) it can be written in multi-line, right?

Reply:
Ah! Sure, I'll wrap the comment. It will occupy two lines.